src/components/main.js

```diff
@@ -15,8 +17,18 @@ export class Main extends Component {
           <Route path="/" element={<LandingPage />} />
           <Route path="signIn" element={<SignIn />} />
           <Route path="SignUp" element={<SignUp />} />
+          <Route path="shop" element={<FilterAndProductListing />} />
+          <Route path="product/:id" element={<ProductDetailsPage />} />
+          <Route
+            path="*"
+            element={
+              <main style={{ padding: "1rem" }}>
+                <p>There's nothing here!</p>
```

Comment:
change message to "We couldn't find the page"

Reply:
Added a page for it